recipe/meta.yaml

```diff
@@ -30,7 +30,7 @@ requirements:
     - {{ compiler('cxx') }}
     - sed               # [unix]
     - openjdk >=8       # [unix]
-    - grpc-cpp          # [unix]
+    - grpc-cpp 1.43         # [unix]
```

Comment:
I thought you wanted a migrator?

Reply:
No, I wanted a migrator for (C++14 abseil + grpc-cpp 1.45, C++17 abseil + grpc-cpp 1.46). 

The build here is to unblock the `libprotobuf=3.20` migration without touching the current `grpc-cpp` pin. After this PR is merged, I would revert the explicit grpc-cpp pin again and do the above mentioned migrator.

Reply:
understood.

Reply:
Yes!